src/coreclr/jit/codegenarmarch.cpp

```diff
@@ -27,19 +27,24 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 //
 // Arguments:
 //    spDelta                 - the value to add to SP. Must be negative or zero.
+//    regTmp                  - an available temporary register.
```

Comment:
can `regTmp` be `REG_NA` when we know that the imm fits?

Reply:
Yes, implementation of `CodeGen::genInstrWithConstant` in codegenarm64.cpp would allow this. I guess I should update the comment saying that.

Reply:
Updated the comment